src/dashboard/components/Dashboard/Catalog/Categories/List/CategoryList.tsx

```diff
@@ -0,0 +1,51 @@
+import { ICategoryLocalized } from "@/types/category";
+import TableContainer from "@mui/material/TableContainer";
+import Paper from "@mui/material/Paper";
+import React from "react";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import Table from "@mui/material/Table";
+import TableCell from "@mui/material/TableCell";
+import TableBody from "@mui/material/TableBody";
+import Link from "next/link";
+
+interface ICategoryListProps {
+  categories: ICategoryLocalized[];
+}
+
+const CategoryList = ({ categories }: ICategoryListProps) => {
+  return (
+    <>
+      <TableContainer component={Paper}>
```

Comment:
Maaaaaaybe we could consider `MUI DataGrid` in the future. It works quite nice. Or something totally custom so that we can play with drag & drop 😊 But this is absolutely fine!

Reply:
Yes, we definitely could, this is just a first version of the category list (there'll definitely be some updates later on)